testdata/corpus.yaml

```diff
@@ -0,0 +1,274 @@
+  - pkg: pbkdf2
+  - pkg: pkcs12/internal/rc2
+  - pkg: ripemd160
+  - pkg: salsa20
+  - pkg: scrypt
+  - pkg: ssh/internal/bcrypt_pbkdf
+  - pkg: tea
+  - pkg: twofish
+  - pkg: xtea
+    # chacha20 -- panic: chacha20: SetCounter attempted to rollback counter
+    # cryptobyte -- panic: unimplemented: reflect.OverflowInt()
+    # salsa20/salsa -- panic: runtime error: index out of range
+    # sha3 -- panic: unimplemented: (reflect.Type).NumMethod()
+- repo: github.com/google/shlex
+- repo: github.com/google/btree
+- repo: github.com/google/der-ascii
+  subdirs:
+  - pkg: cmd/ascii2der
+  - pkg: cmd/der2ascii
+  - pkg: internal
+- repo: github.com/google/hilbert
+- repo: github.com/google/go-intervals
+  subdirs:
+    - pkg: intervalset
+    - pkg: timespanset
+      skipwasi: true # needs timezone stuff
+- repo: github.com/google/okay
+- repo: golang.org/x/text
+  subdirs:
+  - pkg: encoding
+  - pkg: encoding/charmap
+  - pkg: encoding/htmlindex
+  - pkg: encoding/ianaindex
+  - pkg: encoding/japanese
+  - pkg: encoding/korean
+  - pkg: encoding/simplifiedchinese
+  - pkg: encoding/traditionalchinese
+  - pkg: encoding/unicode
+  - pkg: encoding/unicode/utf32
+  - pkg: internal/format
+  - pkg: internal/ucd
+  - pkg: internal/tag
+  - pkg: search
+  - pkg: unicode/rangetable
+  - pkg: message/catalog
+    # 'collate/build', # -- panic: (reflect.Value).Interface: unexported
+    # 'feature/plural', # TestSelect, TestOrdinal, TestCardinal fail
+    # 'internal/catmsg', # TestCodec fails
+    # 'internal/gen/bitfield', # panic: unimplemented: (reflect.Type).Name()
+    # 'number', # fails due to printf %T formatting
+- repo: golang.org/x/image
+  tags: noasm
+  subdirs:
+  # - bmp: needs  _time.startTimer
+  - pkg: ccitt
+  - pkg: colornames
+  - pkg: draw
+  - pkg: font
+  - pkg: font/basicfont
+  - pkg: font/opentype
+  - pkg: font/plan9font
+    skipwasi: true # 2022/01/10 10:43:31 === RUN   TestMetrics: panic: (reflect.Value).Interface: unexported
+  - pkg: math/fixed
+  - pkg: riff
+  - pkg: webp
+  - pkg: tiff
+- repo: github.com/golang/geo
+  subdirs:
+  - pkg: r1
+  - pkg: r2
+  - pkg: r3
+  - pkg: s1
+    #  's2' -- reflect.DeepEqual() -> MapKeys
+- repo: github.com/golang/groupcache
+  subdirs:
+  - pkg: consistenthash
+  - pkg: lru
+- repo: github.com/armon/go-radix
+- repo: github.com/armon/circbuf
+- repo: github.com/VividCortex/gohistogram
+- repo: github.com/cespare/xxhash
+  tags: appengine
+- repo: gonum.org/v1/gonum
+  tags: noasm
+- repo: gonum.org/v1/gonum
+  tags: noasm appengine
+  subdirs:
+  - pkg: blas/blas32
+  - pkg: blas/cblas64
+  - pkg: blas/cblas128
+  - pkg: cmplxs/cscalar
+  - pkg: dsp/window
+  - pkg: floats/scalar
+  - pkg: integrate
+  - pkg: integrate/quad
+  - pkg: internal/cmplx64
+  - pkg: internal/testrand
+  - pkg: lapack/gonum
+    skipwasi: true # takes too long
+    slow: true
+  - pkg: mathext
+  - pkg: mathext/prng
+  - pkg: optimize/convex/lp
+    skipwasi: true # takes too long
```

Comment:
Need a `slow: true` here too or is it fast (enough) on native?

Reply:
Fast enough on native